docs/stores/vuex/vuex.md

```diff
@@ -428,3 +428,17 @@ Here's the complete example:
 - [Vuex](https://vuex.vuejs.org/) - Vuex official docs.
 
 - Additional examples - [Vuex](https://github.com/vuejs/vuex#examples)
+
+## Project Tasks
+
+You might have notice the emitter patter to be useful, but also, to be hard to maintain and implement as your application gets complex. Let's implement a  Vuex patter and see hot it comes into place:
```

Comment:
there's a typo in:
Vuex patter and see **how** it comes into place:

Reply:
**pattern** is misspelled twice.